github/HookDelivery.py

```diff
@@ -44,86 +44,50 @@ def __repr__(self) -> str:
 
```

Comment:
definition of attributes is missing before `__repr__`:

```python
    _request: Attribute[HookDeliveryRequest]
    _response: Attribute[HookDeliveryResponse]
```

Reply:
It's in the method initAttributes